react/SearchWrapper.js

```diff
@@ -1,63 +1,64 @@
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-import { Helmet, withRuntimeContext } from 'vtex.render-runtime'
+import React, { Fragment, useMemo } from 'react'
+import { Helmet, useRuntime } from 'vtex.render-runtime'
 
-import DataLayerApolloWrapper from './components/DataLayerApolloWrapper'
 import { capitalize } from './utils/capitalize'
+import useDataPixel from './hooks/useDataPixel'
+
 const APP_LOCATOR = 'vtex.store'
 
-class SearchWrapper extends Component {
-  static propTypes = {
-    /** Route parameters */
-    params: PropTypes.shape({
-      category: PropTypes.string,
-      department: PropTypes.string,
-      term: PropTypes.string,
-    }),
-    /** Render runtime context */
-    runtime: PropTypes.shape({
-      page: PropTypes.string.isRequired,
-      account: PropTypes.any,
-      getSettings: PropTypes.func,
-    }),
-    /** Search query result */
-    searchQuery: PropTypes.object.isRequired,
-    /** Component to be rendered */
-    children: PropTypes.node.isRequired,
+const pageCategory = (products, params) => {
+  if (!products || products.length === 0) {
+    return 'EmptySearch'
   }
+  const { category, term } = params
+  return term ? 'InternalSiteSearch' : category ? 'Category' : 'Department'
+}
 
-  pageCategory = products => {
-    if (!products || products.length === 0) {
-      return 'EmptySearch'
-    }
-    const { category, term } = this.props.params
-    return term ? 'InternalSiteSearch' : category ? 'Category' : 'Department'
+const getPageEventName = (products, params) => {
+  if (!products || products.length === 0) {
+    return 'otherView'
   }
+  const category = pageCategory(products, params)
+  return `${category.charAt(0).toLowerCase()}${category.slice(1)}View`
+}
 
-  getPageEventName = products => {
-    if (!products || products.length === 0) {
-      return 'otherView'
-    }
-    const pageCategory = this.pageCategory(products)
-    return `${pageCategory.charAt(0).toLowerCase()}${pageCategory.slice(1)}View`
-  }
+const getTitleTag = (titleTag, storeTitle, term) => {
```

Comment:
//prettier-ignore	to make the manual identation

Reply:
this function is outside the component definition, the indentation is correct